lambdanotes/src/components/SideBar.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import './SideBar.css';
+
+import logo from '/Users/josephbradley/Desktop/practice-dir/front-end-project-week/lambdanotes/src/assets/lambda.png';
```

Comment:
Be careful using specific paths like this. It will only work on your computer since others will note have the same path.